.github/workflows/deploy.yml

```diff
@@ -15,10 +15,13 @@ jobs:
       users: 2
       spawn-rate: 2
       run-time: 11s
+      run_performance_tests: true
+      run_e2e_tests: true
     if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
dependabot doesnt have access to the secrets (E2E PAT) so I think this secret will need adding to dependabot variables (https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-encrypted-secrets-for-dependabot)

Reply:
or just move that line up so e2e and performance tests do not run on dependabot PRs :)

Reply:
Now moved this if into the shared workflow, so removed from here